Tests/eppo/SharderTests.swift

```diff
@@ -0,0 +1,25 @@
+import XCTest
+
+@testable import eppo_flagging
+
+class SharderTests: XCTestCase {
+    
+    func testMD5Sharder() {
+        let sharder = MD5Sharder()
+        let input = "testInput"
+        let totalShards = 10
+        let shard = sharder.getShard(input: input, totalShards: totalShards)
+        
+        XCTAssertGreaterThanOrEqual(shard, 0)
+        XCTAssertLessThan(shard, totalShards)
+    }
```

Comment:
Added some MD5 sharder tests to JS-commons here, might be nice to copy-paste to ensure they are the same:

https://github.com/Eppo-exp/js-client-sdk-common/pull/52/files#diff-ea83fb75e9f26161475fd7e79c9bd4eb4292dc61b8b02161d43baa036f5d6140